src/vs/workbench/browser/labels.ts

```diff
@@ -59,6 +60,11 @@ export interface IResourceLabelOptions extends IIconLabelValueOptions {
 	 * Will take the provided label as is and e.g. not override it for untitled files.
 	 */
 	readonly forceLabel?: boolean;
+
+	/**
+	 * Uses the provided icon instead of deriving a resource icon.
+	 */
+	readonly codIcon?: ThemeIcon;
```

Comment:
"codicon" is the right casing. This looks like an icon of a fish 😉 

It might be better to just call this `icon` though?

Reply:
Ohh thanks :D Jup, `icon` should be good enough